docker/atlassian.yml

```diff
@@ -11,7 +11,7 @@ services:
         image: ghcr.io/ls1intum/artemis-jira:9.4.3
         pull_policy: always
         volumes:
-          - artemis-jira-data:/var/atlassian/application-data/jira
+            - artemis-jira-data:/var/atlassian/application-data/jira
```

Comment:
Is this necessary?

Reply:
Not really (YAML does not differentiate), but makes it uniform to the specification of the other volumes defined in the file.

Reply:
It's a good idea to revert it then, so we don't have unrelated changes in the scope of single PR